js/components/widgets/stackViewer/StackViewerComponent.js

```diff
@@ -443,18 +443,21 @@ define(function (require) {
               success: function (data) {
                 result = data.trim().split(':')[1].trim().split(' ');
                 if (result !== '') {
-                  for (j in result) {
-                    if (result[j].trim() !== '') {
-                      var index = Number(result[j]);
-                      if (i !== 0 || index !== 0) { // don't select template
-                        if (index == 0) {
-                          if (!shift) {
-                            that.state.objects.push(that.state.label[i]);
-                          }
-                        } else {
-                          if (typeof that.props.templateDomainIds !== 'undefined' && typeof that.props.templateDomainNames !== 'undefined' && typeof that.props.templateDomainIds[index] !== 'undefined' && typeof that.props.templateDomainNames[index] !== 'undefined' && that.props.templateDomainNames[index] !== null) {
-                            that.state.objects.push(that.props.templateDomainNames[index]);
-                            break;
+                  var currentPosition = that.renderer.plugins.interaction.mouse.getLocalPosition(that.stack);
```

Comment:
This doesn't seem related to the current scope of the pr. Is this here for a reason or it's just coming form local merging?

Reply:
local merge as you pointed out, as written in the previous message I added robbie as reviewer to check this is fine since he was working with this.

Reply:
Can be that this is also part of another pr? This always happens to me also when I'm working on a project and yet I have to find a clean way to get rid of additional code. Doing a rebase on the remote dev could be a solution but the history is stays littered if there was a push. My best solution for now is to start a new feature branch from the last dev and add only the code I need there for the pr.